edalize/templates/apicula/apicula-makefile.j2

```diff
@@ -0,0 +1,21 @@
+#Auto generated by Edalize
+
+TARGET   := {{ name }}
+CST_FILE := {{ cst_file }}
+NEXTPNR_OPTIONS     := {{ nextpnr_options|join(' ') }}
+
+all: $(TARGET).fs
+
+%.json: %.mk
+	$(MAKE) -f $<
+%.pack: %.json
+	nextpnr-gowin -l next.log --device $(DEVICE)  $(NEXTPNR_OPTIONS) --cst $(CST_FILE) --json $? --write $@
```

Comment:
For consistency with trellis back end,  in the line 12, --device should be defined in NEXTPNR_OPTIONS : [--device,GW1N-LV1QN48C6/I5]. Otherwise in the .core file the first parameter must be the device part number or nextpnr-gowin will stop with error. Furthermore, $(DEVICE) is an empty string.

Reply:
Now that we can pass the same device identifier to both nextpnr and gowin_pack it's better to have an explicit device (or maybe part) option. We should also error out if the user didn't set this option